testing_and_setup/compass/ocean/global_ocean/QU240wISC/init/define_vertical_grid.py

```diff
@@ -0,0 +1 @@
+../../../scripts/vertical_grid/define_vertical_grid_64_layers.py
```

Comment:
I think links like this are needed for a lot more test cases, right?
* CUSP12
* CUSP8
* QU60
* SO60to10wISC
* SOQU60to15 (is this even a configuration we want to keep?)